src/sentry/preprod/tasks.py

```diff
@@ -277,13 +278,25 @@ def _assemble_preprod_artifact_size_analysis(
             pass  # Ignore cleanup errors
         raise Exception(f"PreprodArtifact with id {artifact_id} does not exist")
 
+    size_analysis_results = SizeAnalysisResults.parse_raw(assemble_result.bundle_temp_file.read())
+    install_size = (
+        size_analysis_results.treemap.total_install_size if size_analysis_results.treemap else None
```

Comment:
For iOS I added a couple top-level fields for this since the install/download size had a slight discrepancy with extra data that's not really attributable to treemap nodes, e.g. some padding/alignment. See: https://github.com/getsentry/launchpad/blob/main/src/launchpad/size/models/apple.py#L45-L46

I could remove these fields and instead manually set `total_install_size` and `total_download_size` rather than calculate it from the sum of its children, wdyt?